src/sage/rings/lazy_series.py

```diff
@@ -6065,6 +6068,44 @@ def polynomial(self, degree=None, names=None):
             return R(self[0:m])
         return R.sum(self[0:m])
 
+    def add_bigoh(self, prec):
+        r"""
+        Return the power series of precision at most ``prec`` obtained by
+        adding `O(q^\text{prec})` to `f`, where `q` is the variable(s).
```

Comment:
Extraordinarily minor: perhaps:
```
adding `O(q)^\text{prec}` to `f`, where `q` is the (list of) variable(s).
```
is more correct.  (Note that sage seems to prefer `O(q,t)^3`.)

Reply:
Done (I just copied it from the other methods, but I agree that this is better.)